app/models/concerns/lending_period.rb

```diff
@@ -0,0 +1,63 @@
+module LendingPeriod
+
+  extend ActiveSupport::Concern
+
+  included do
+    property :lending_period, predicate: ::RDF::Vocab::SCHEMA.eligibleDuration, multiple: false do |index|
+      index.as :stored_sortable
+    end
+
+    after_initialize :set_lending_period
```

Comment:
I'm inclined to name this `default_lending_period` for collections and `lending_period` for media objects to follow the current pattern.  This would break the DRYing up done here in this concern.

I wonder if the initialization of the lending period is necessary or if when it is being called it would fall back to the global default.  Maybe this could be done in an accessor on the media_object:
```
def lending_period
  self.lending_period || Settings.controlled_digital_lending.default_lending_period
end
```
It might be that the media object will always get initialized from the collection or global default and this accessor override is unnecessary.

I'm also wondering which would be better: storing the ISO8601 string in the property or the integer.  It seems like there is a decent amount of code for casting back and forth so wondering if that would be easier if we store the string form.  Any thoughts on this?

Another thought is making a subclass of ActiveSupport::Duration which at a minimum could implement the `display_lending_period` helper possibly as a `to_s` override.